Homework/Homework_11/Library.py

```diff
@@ -0,0 +1,69 @@
+class Book:
+    def __init__(self, title, author, pages, ISBN):
+        self.title = title
+        self.author = author
+        self.pages = pages
+        self.ISBN = ISBN
+        self.is_reserved = False
+        self.is_taken = False
+        self.username_take = None
+
+
+class User:
+    def __init__(self, username):
+        self.username = username
+
+    def reserve_book(self, book):
+        if book.is_reserved:
+            if book.username_take == self.username:
+                return 'You are already reserved this book'
+            return 'Book is already reserved'
+        if book.is_taken:
+            if book.username_take == self.username:
+                return 'You are already taken this book'
+            return 'Book is already taken'
```

Comment:
Looks really good, but it is better to return `bool` instead of `str`:

```suggestion
    def reserve_book(self, book):
        if book.is_reserved:
            if book.username_take == self.username:
                print('You are already reserved this book')
                return False
            print('Book is already reserved')
            return False
        if book.is_taken:
            if book.username_take == self.username:
                print('You are already taken this book')
                return False
            print('Book is already taken')
            return False
```

Note: this applicable for whole program

Reply:
Fix